src/Notify/README.md

```diff
@@ -0,0 +1,167 @@
+# Symfony UX Notify
+
+Symfony UX Notify is a Symfony bundle integrating realtime native notifications in Symfony applications using [Mercure](https://mercure.rocks/).
+It is part of [the Symfony UX initiative](https://symfony.com/ux).
+
+## Installation
+
+Symfony UX Notify requires PHP 7.2+ and Symfony 5.3+.
+
+Install this bundle using Composer and Symfony Flex:
+
+```sh
+composer require symfony/ux-notify
+
+# Don't forget to install the JavaScript dependencies as well and compile
+yarn install --force
+yarn encore dev
+```
+
+Also make sure you have at least version 3.0 of [@symfony/stimulus-bridge](https://github.com/symfony/stimulus-bridge) in your `package.json` file.
+
+## Usage
+
+To use Symfony UX Notify you must have a [running Mercure server](https://symfony.com/doc/current/mercure.html#running-a-mercure-hub).
+
+Then, inject the `NotifierInterface` service and send messages on the `chat/mercure` channel.
+
+```php
+// ...
+use Symfony\Component\Notifier\Notification\Notification;
+use Symfony\Component\Notifier\NotifierInterface;
+
+class AnnounceFlashSalesCommand extends Command
+{
+    protected static $defaultName = 'app:flash-sales:announce';
+    private $notifier;
+
+    public function __construct(NotifierInterface $notifier)
+    {
+        parent::__construct();
+
+        $this->notifier = $notifier;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $this->notifier->send(new Notification('Flash sales has been started!', ['chat/mercure']));
+
+        return 0;
+    }
+}
+```
+
+Finally, native notifications could be displayed using the `notify` Twig function:
+
+```twig
+{{ stream_notifications(['/my/topic/1', '/my/topic/2']) }}
+{{ stream_notifications() }}
+
+{# Calling notify without parameter will fallback to the following unique topic: 'https://symfony.com/notifier' #}
+```
+
+### Using another Mercure hub
+
+By default, the `stream_notifications` is streaming the default Mercure hub. If you wanna change that behavior, you can use a compiler pass:
+
+```php
+// src/DependencyInjection/UseCustomNotifyMercureHubCompilerPass.php
+
+namespace App\DependencyInjection;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Reference;
+
+final class UseCustomNotifyMercureHubCompilerPass implements CompilerPassInterface
+{
+    public function process(ContainerBuilder $container): void
+    {
+        $container->getDefinition('notify.twig_extension')
+            ->replaceArgument(0, new Reference('mercure.hub.custom'));
```

Comment:
If this is a common need, I'd suggest creating a config for it, so one could change easily the Mercure hub linked to this service:
```yaml
notify:
    mercure_hub: mercure.hub.default # default service id
```

Reply:
That's a great idea indeed! :+1: